src/AzureDevOpsPolicyConfigurator/Logic/PolicyExecuter.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using System.Collections.Generic;
+using AzureDevOpsPolicyConfigurator.Data;
+using AzureDevOpsPolicyConfigurator.Interfaces;
+using Microsoft.TeamFoundation.Policy.WebApi;
+using Microsoft.TeamFoundation.SourceControl.WebApi;
+
+namespace AzureDevOpsPolicyConfigurator.Logic
+{
+    /// <summary>
+    /// PolicyExecuter class.
+    /// </summary>
+    internal class PolicyExecuter : PolicyExecuterBase
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="PolicyExecuter"/> class.
+        /// </summary>
+        /// <param name="serializer">Json serializer</param>
+        /// <param name="reader">File reader</param>
+        /// <param name="connectionProvider">Connection provider</param>
+        public PolicyExecuter(IJsonSerializer serializer, IFileReader reader, IConnectionProvider connectionProvider)
+            : base(serializer, reader, connectionProvider)
+        {
+        }
+
+        /// <summary>
+        /// Creates a policy in TFS.
```

Comment:
Use AzureDevOps instead of TFS, as on-prem product is now also called Azure DevOps Server